guide/getting-started/open-design.md

```diff
@@ -50,67 +50,66 @@ Illustration sources
-- continuously learn independently and from each other
-- rely on our peer group for free and communal validation of quality
-- are accountable for our actions and have vested interest in the groups' actions and goals
+- collaborate without competing
+- share our process and work in a transparent way
+- make it easy for others to build on our work and ideas
+- aim for inclusivity
+- approach problems with an open-mind
+- embrace multidisciplinary feedback
+- rely on our peers for quality assurance and validation
+- fulfill the open design group’s goals through action
+
+
+We value:
 
-And we value:
+- moral and responsible choices
+- products that solve real problems and address real needs
+- usability accessibility for the broadest possible range of people, locations, and backgrounds
+- highly functional aesthetic elements that influence both how people feel and think
 
-- design ethics to make moral and responsible choices
-- human design to create products for real needs and desires
-- usability to create products that allow users to solve their problems effectively
-- accessibility to make products usable by people with the broadest possible range of abilities, in the widest possible range of situations
-- aesthetics, which has a highly functional element and also influences how people feel and think
 
-## Getting started
+## Getting
+
+If you are new to open design for bitcoin, which many are, it can be tricky to find a place to start. Not only is the tech complex, but there are few educational resources available. The  [Contribute page]({{ '/contribute' | relative_url }}) is a good starting point. These tips can also help you find your footing:
 
-If you are completely new to open design, it can feel tricky to find a starting point in the Bitcoin ecosystem. Not only is the technology complex, but there are also few design resources available. A good starting point is the [Contribute]({{ '/contribute' | relative_url }}) page, and here are a few extra tips to help you find your way around in general:
 
 - Join the [Bitcoin Design Community]({{site.slack_invite_url}}) on Slack and see what others are discussing. A good way to get started is by asking questions and joining conversations that interest you. Our [Community Calls](https://github.com/BitcoinDesign/Meta/issues?q=is%3Aissue+is%3Aopen+Community+Call) and [Design Review](https://github.com/BitcoinDesign/Meta/issues?q=is%3Aissue+is%3Aopen+Design+Review+Call+) calls are also great ways to get active.
-- Download a Bitcoin wallet and experiment with small amounts. You can also use Testnet, which allows you to use all the features, but with worthless test bitcoin. Pay attention to the experience and come up with ways to improve it.
-- Redesign a Bitcoin application of your choice. Even as a brief exercise, a redesign is a great way to develop a deeper understanding of how something works and why certain decisions were made.
```

Comment:
Why not recommend people to do design exercises? I personally find these super helpful and educational.

Reply:
Removed by mistake, accidentally duplicated the paragraph and overlooked this. Nice catch fixed in https://github.com/BitcoinDesign/Guide/pull/290/commits/e1e2978ab41c6df2eaef38bbd512dbcceda0697b